docs/users/installation.md

```diff
@@ -0,0 +1,225 @@
+---
+id: installation
+title: Installation
+---
+
+To run scalafix on your project, you must first install the Scalafix sbt plugin
+or command line interface. Currently, Scalafix does not provide any IDE
+integrations.
+
+| Scalafix Version | Scalameta(SemanticDB) Version | Scala Version           |
+| ---------------- | ----------------------------- | ----------------------- |
+| 0.5.10           | 2.1.7                         | 2.11.12 / 2.12.4        |
+| @VERSION@        | @SCALAMETA@                   | @SCALA211@ / @SCALA212@ |
+
+## sbt
+
+The quickest way to get started is to run a **syntactic** rule like
+`ProcedureSyntax`
+
+```scala
+// project/plugins.sbt
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "@VERSION@")
+
+// sbt shell
+> myproject/scalafix ProcedureSyntax      // run rule on main sources
+> myproject/test:scalafix ProcedureSyntax // run rule on test sources
+```
+
+To run **semantic** rules like `RemoveUnusedImports` you need additional build
+configuration.
+
+```scala
+// build.sbt
+lazy val myproject = project.settings(
+  scalaVersion := "@SCALA212@", // or @SCALA211@
+  addCompilerPlugin(scalafixSemanticdb),
+  scalacOptions ++= List(
+    "-Yrangepos",          // required by semanticdb-scalac compiler plugin
+    "-Ywarn-unused-import" // required by `RemoveUnusedImports` rule
+  )
+)
+
+// sbt shell
+> myproject/scalafix RemoveUnusedImports      // run rule on main sources
+> myproject/test:scalafix RemoveUnusedImports // run rule on test sources
+```
+
+Notes.
+
+- The `semanticdb-scalac` compiler plugin produces
+  [SemanticDB](https://github.com/scalameta/scalameta/blob/master/semanticdb/semanticdb3/semanticdb3.md)
+  files in the target directory `META-INF/semanticdb/`. These files are used by
+  Scalafix to understand symbols and types in Scala source code.
+- Beware that the `semanticdb-scalac` compiler plugin adds around 7-25% overhead
+  to compilation. It's recommended to provide generous JVM memory and stack
+  settings in `.jvmopts`:
+
+      -Xss8m
+      -Xms1G
+      -Xmx4G
+
+### Verify installation
+
+To verify the installation, check that scalacOptions and libraryDependencies
+contain the values below.
+
+```scala
+> show scalacOptions
+[info] * -Yrangepos
+> show libraryDependencies
+[info] * org.scalameta:semanticdb-scalac:@SCALA212@:plugin->default(compile)
+```
+
+### Example project
+
+For a minimal example project using sbt-scalafix, see the
+[scalacenter/scalafix-sbt-example](https://github.com/scalacenter/scalafix-sbt-example)
+repository.
+
+```scala
+git clone https://github.com/olafurpg/scalafix-sbt-example
```

Comment:
Should be under `scalacenter/scalafix-sbt-example`.

In the front page we have gitter8, We should stick with only one example. I prefer `git clone` since it's more simple.

Reply:
Good catch, the repo is already under `scalacenter/`, was a bad link.

I agree confusing to advertise two different examples. The front page links to the template for rule authors while this is an example for end users. It's best to remove the reference to `scalafix.g8` on the landing page and use a better image for "Extensible". It's not possible to have git clone workflow for rule authors since filenames dependent on the project name.